### ISSUE TITLE
I open the link but it does not go directly to the app

### ISSUE THREAD

**Issue by ImLyfest** [swaggyP36000/TrollStore-IPAs#68]
Yesterday I opened an issue but I think I was wrong, Well the problem is that I open a link from some search for instagram or twitter but it does not send me directly to the app it opens in the same safari, I just updated twitter and instagram BH the two apps by the way

**Comment by ArianaMinajj** [comment]
Hi there,

I understand your concern. I actually addressed this in the previous issue you opened. Deep links can be utilized as Safari extensions, but only through sideloading. At the moment, it's not feasible to implement direct deep linking with TrollStore. (I might be wrong)

If you have any more questions or need further assistance, feel free to ask.

**Comment by ImLyfest** [comment]
Yes it has, in previous updates it worked for me, right now I found one for instagram and it is in the latest version with deep link

**Comment by ArianaMinajj** [comment]
Apologies for any confusion. While I'm not personally a user of trollstore, I want to note that there's a possibility of me providing inaccurate information.

Nonetheless, I've conducted a self-test, and I can confirm that deep links are functional in the case of [InstaRocketBH](https://github.com/swaggyP36000/TrollStore-IPAs/releases?q=InstaRocketBH+&expanded=true). However, when it comes to the latest version of BHTwitter, deep links do not appear to be working.

Please feel comfortable leaving this issue opened until deep links become accessible for both applications.

**Comment by ImLyfest** [comment]
I appreciate the hard work you do, I'll be ready when you let me know, You don't have to apologize, In the meantime I'll be using an older version of BHTwitter uploaded by you of course 😀

**Comment by swaggyP36000** [comment]
this is a known issue having more to do with the new versions of TrollStore. for some reason the deeplink integrations got more flaky with new versions of TrollStore. there's no convenient way around this unfortunately

Some devs, for example the dev of BHTwitter, made a trollstore specific version that has deep link support, etc. you can always reach out to tweak devs but there's a good chance a lot of them don't actually use TrollStore, their apps just happen to work fine on TrollStore outside the link support stuff

**Comment by Genxster1998** [comment]
deeplink requires original appid and bundleid as well as certain entitlements and bundled extensions. Sometimes Dev signs app with private signature and wrong entitlement file and strips out extensions which strip out important os integration info.
